api/types/crds/skupper_attached_connector_anchor_crd.yaml

```diff
@@ -19,6 +19,8 @@ spec:
                   type: string
                 routingKey:
                   type: string
+                exposeTargetByName:
```

Comment:
I propose "exposePodsByName".  I find it more direct and explanatory.

Reply:
Done!